examples/sushi/config.py

```diff
@@ -29,9 +29,17 @@
 model_defaults_iceberg = ModelDefaultsConfig(**defaults, storage_format="iceberg")
 
 
-# An in memory DuckDB config.
+# A DuckDB config, in-memory by default.
```

Comment:
there's already a local_config, should we remove that or if not, why do have it here too?

Reply:
good catch - removed in [7505ace](https://github.com/TobikoData/sqlmesh/pull/2938/commits/7505ace0827269ae4293e369c4eb4236ba315705)